Create footer component for new docs site

Ray2126 opened the pull request:
Based on this Figma component: https://www.figma.com/file/IgRlQWJTyGoZhlF4fVKD6B/Tailwind-Centrapay-Docs?node-id=317%3A4398&t=UroKj5xGKWgGkgYT-0

HTML taken from: https://tailwindui.com/components/marketing/sections/footers

On desktop:

<img width="2047" alt="image" src="https://user-images.githubusercontent.com/50732795/212585727-11e03afc-825c-489c-8185-dd147642cdf8.png">

On mobile: 

<img width="414" alt="image" src="https://user-images.githubusercontent.com/50732795/212585772-6f886344-1950-4179-8b2c-48ff4bb292af.png">

MeganSteenkamp commented on components/Footer.vue:
```diff
@@ -1,6 +1,58 @@
+import facebookLogo from './icons/FacebookLogo.vue';
+import instagramLogo from './icons/InstagramLogo.vue';
+import twitterLogo from './icons/TwitterLogo.vue';
+import linkedInLogo from './icons/LinkedInLogo.vue';
```
Instead of individually importing each logo I was wondering about globally importing our icons in `nuxt.config.js`:
```
components: {
    'dirs': [
      {
        'path': '~/components/icons',
        'global': true
      },
      '~/components'
    ]
  },
  ```
  
  Then you can just declare the logo as `facebook-logo` etc.

MeganSteenkamp commented on components/Footer.vue:
```diff
@@ -1,6 +1,58 @@
+            <span class="sr-only">{{ item.name }}</span>
+            <component
+              :is="item.logo"
+              class="h-6 w-6"
```
You can use the `icon-lg` class here 😄

Ray2126 commented:
Matthias has asked for:

- The top border to be removed
- Remove Facebook and Instagram links, add GitHub
- The year to be dynamic

Ray2126 commented:
### New footer

Changes:
- Top border has been removed
- Removed Instagram and Facebook social links
- Added Github link
- Year is now dynamic
- Icons are now auto imported

Desktop:
<img width="2047" alt="image" src="https://user-images.githubusercontent.com/50732795/212594637-b19f718e-846b-4d0c-aa18-30fb26c06473.png">

Mobile: 
<img width="414" alt="image" src="https://user-images.githubusercontent.com/50732795/212594790-fd1c219f-dadd-4037-8269-ec6517a58e79.png">
